source/includes/fundamentals/code-snippets/records/CodecRecordExample.java

```diff
@@ -0,0 +1,37 @@
+package fundamentals.codecs.records;
+
+import static com.mongodb.MongoClientSettings.getDefaultCodecRegistry;
+import static org.bson.codecs.configuration.CodecRegistries.fromRegistries;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.bson.codecs.configuration.CodecRegistry;
+
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+
+
+public class CodecRecordExample {
+    public static void main(String args[]) {
+        CodecRegistry codecRegistry = fromRegistries(getDefaultCodecRegistry());
```

Comment:
This seems like a no-op.  What's the reason for including it?

Reply:
Oops, you're right -- I'll remove it. Initially, I was creating a custom Codec to mirror the POJO page, but decided to pare back the scope. Will address your related comments as well:
https://github.com/mongodb/docs-java/pull/280#discussion_r968736992
https://github.com/mongodb/docs-java/pull/280#discussion_r968738042
https://github.com/mongodb/docs-java/pull/280#discussion_r968738856